style/style.css

```diff
@@ -0,0 +1,1523 @@
+*{
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-decoration: none;
+    font-family: Arial, Helvetica, sans-serif;
+    }
+    .wrapper{
+        max-width: 1200px;

```

Comment:
there is no need to do it. That is the result 
![image](https://user-images.githubusercontent.com/47325983/104042302-6050bb00-51eb-11eb-8631-6b25f049b1a6.png)

Reply:
so if the resolution of a screen is wider than 1200px, users see empty space on the right side